pkgs/development/python-modules/youtube-search-python/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, isPy27, fetchPypi, httpx }:
+
+buildPythonPackage rec {
+  pname = "youtube-search-python";
+  version = "1.4.7";
+
+  disabled = isPy27;
```

```suggestion
  disabled = pythonOlder "3.6";;
```

see https://github.com/alexmercerind/youtube-search-python/blob/main/setup.py#L27. Just to be on the safe site.